dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -304,6 +304,9 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 		CAPIArticle.config.switches.automaticFilters &&
 		CAPIArticle.availableTopics;
 
+	const shouldReserveMerchSpace =
+		CAPIArticle.config.switches.merchandisingMinHeight;
```

Comment:
I think this will condition will just check whether the server-side test is enabled, rather than whether the user has been placed in a variant for the experiment.

I can't remember the exact syntax but I think it's something like...

```typescript
CAPIArticle.config.abTests?. merchandisingMinHeight === 'variant';
```

Reply:
That makes sense. I'll give that a try. Thanks!